source/jobs/recover-jobs.txt

```diff
@@ -0,0 +1,77 @@
+.. _rm-sync-jobs-recoverability:
+
+==================
+Recover a Sync Job
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can attempt to recover continuous (CDC) sync jobs 
+that have failed within the preceding 24-hour period. 
+The procedure for recovering a CDC sync job differs, 
+depending on whether you're using the embedded or Kafka deployment 
+mode. For details on each job type, see :ref:`rm-sync-jobs`.
+
+About this Task
+---------------
+
+CDC sync job recovery depends on the type of deployment 
+mode your using. The following table summarizes the modes:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Mode
+     - Description
+
+   * - Embedded Mode
+     - Sync jobs that are running on a :ref:`local machine <local-machine-install>` or 
+       a :ref:`unattended server <unattended-server>`.
+
+   * - Kafka Mode
+     - Sync jobs that are running with :ref:`Kafka support <kafka-intro>`.
+
+Task
+----
+
+Click the tab to display the recovery procedure 
+for each deployment mode:
+
+.. tabs::
+
+   .. tab:: Embedded Mode
+      :tabid: embedded-mode
+
+      For embedded mode recovery, the process to restart
+      a sync job is manually trigger. Navigate to the 
+      :guilabel:`Data Migration` tab and click 
+      :guilabel:`Resume` on the failed sync job. 
+
+      Examples of embedded mode recoverable scenarios include:
+
+      - Unexpected application failure
+      - Operating system restarts
```

Comment:
(suggestion)
What do you think about adding this detail into the "About this Task" section as a third column. So something like 
Mode | Description | Recovery Scenarios. 

Reasoning is that I think you'd want this detail up front rather than during the task itself.

Reply:
Sure updated.